docs/protocol.md

```diff
@@ -727,6 +728,16 @@ For example you may want to keep some in memory state, i.e. cache, aggregation.
 You can use read the `sk` and forward the call request to a specific process
 that has ownership for the shard key.
 
+### Transport Header `cr` -- Custom routing
+
+The custom routing request header must be set to a valid service name.
+
+When the custom routing header is set we will route to the service name
+in the `cr` header instead of the service name in the call request.
+
+It is expected that the service in the `cr` header is capable of
```

Comment:
This makes it sound even more similar to `sk`. I'm having trouble seeing the difference between these two.